generic/tests/netperf.py

```diff
@@ -148,8 +153,19 @@ def _pin_vm_threads(vm, node):
     vm.verify_alive()
     login_timeout = int(params.get("login_timeout", 360))
 
-    session = vm.wait_for_login(timeout=login_timeout)
+    server_ip = vm.wait_for_get_address(0, timeout=5)
+    if len(params.get("nics", "").split()) > 1:
+        vm.wait_for_serial_login(timeout=30, restart_network=True)
```

Comment:
What does this do?

Reply:
Since, when there are two ethernets on vm, except for changing the two of if-ethx to force up, there is only one or none ethernet could get the ip when bootup, so we need to force let vm to get ip before testing.

Reply:
OK, then please close the session which is created by this procedure.